prisma/schema.prisma

```diff
@@ -86,6 +87,19 @@ model ApiToken {
   updatedAt        DateTime          @updatedAt
 }
 
+model Erc20Token {
```

Comment:
@jumperdevweb3  can't we just put data such as name, symbol, decimals and supply into the payload in the SmartContract model? or even add those columns as they are pretty generic?